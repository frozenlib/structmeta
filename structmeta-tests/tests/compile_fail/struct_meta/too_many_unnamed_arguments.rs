#[structmeta_tests::attr_required_unnamed2(10, 20, 30)]
fn my_func() {}

fn main() {}
