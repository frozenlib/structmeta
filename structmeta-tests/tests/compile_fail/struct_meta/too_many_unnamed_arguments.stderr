error: too many arguments.
 --> $DIR/too_many_unnamed_arguments.rs:1:52
  |
1 | #[structmeta_tests::attr_required_unnamed2(10, 20, 30)]
  |                                                    ^^
